### ISSUE TITLE
Add a sample app for folding interface which is now supported in flutter 2.00

### ISSUE THREAD

**Issue by i-anuragmishra** [flutter/samples#750]
Flutter 2.00 supports foldable phones.  we should add a app showing foldable interface in samples.

**Comment by i-anuragmishra** [comment]
@RedBrogdon can I be assigned this?

**Comment by abd99** [comment]
Hello @i-anuragmishra,

Thanks for show interest in contributing to the repo!

As per my knowledge, foldable support hasn't been merged into the SDK yet.
For more details, visit: [Announcing Flutter support for foldable devices
](https://devblogs.microsoft.com/surface-duo/flutter-dual-screen-foldable/)

With that said, I too feel that adding a sample on this topic would be a valuable addition to the repo. I guess we'll have to wait until it lands into beta/stable and the team approves the need for this sample.

**Comment by i-anuragmishra** [comment]
alright @abd99 closing this issue for now.

**Comment by domesticmouse** [comment]
I'm going to re-open this ticket as a signifier of future required work, to help with prioritising work going forward. Please star this issue if foldable device support samples is important to you =)

**Comment by i-anuragmishra** [comment]
Agreed @domesticmouse ....  I feel we can star it as enhancements @abd99 and may be include a experimental sample in future.
